src/data/faq.json

```diff
@@ -2838,101 +2816,6 @@
-                                ]
-                            },
-                            {
-                                "title": "Is the data of the users of the Corona-Warn-App still secure?",
-                                "anchor": "check_in_luca_safe",
-                                "active": false,
-                                "textblock": [
-                                    "Yes. Joint use only applies to the QR codes used for event registration and check-in. Corona-Warn-App and luca app continue to be different offerings with different objectives. Scanning a luca-QR-Code does not cause any additional data to be collected.",
-                                    "The joint use of QR codes was carefully prepared. All security-related aspects were worked out and checked in close cooperation with both the BfDI and the BSI.",
-                                    "In addition, the Corona-Warn-App continues to operate in a very data-saving manner."
-                                ]
-                            },
-                            {
-                                "title": "What data is transferred from the CWA to luca?",
-                                "anchor": "check_in_luca_one",
-                                "active": false,
-                                "textblock": [
-                                    "No data is transferred from the Corona-Warn-App to the luca app."
-                                ]
-                            },
-                            {
-                                "title": "Do we now only need one app for event registration?",
-                                "anchor": "check_in_luca_both",
-                                "active": false,
-                                "textblock": [
-                                    "The Corona-Warn-App and luca are based on different approaches and have different goals.",
-                                    "The Corona-Warn-App enables a quick warning of people based on the peer-to-peer principle. This warning is done pseudonymously via the Corona-Warn-App itself.",
-                                    "The luca app, on the other hand, is mainly used to digitally collect contact data of the participants of an event. In the event of infection, the person who tests positive for SARS-CoV-2 transfers his or her contact data to the responsible health department. This department then contacts all persons who have scanned the same QR code.",
-                                    "In the respective state ordinances on infection protection, it is stipulated whether and how contact tracking must take place at events."
-                                ]
-                            },
-                            {
-                                "title": "Do users need to install both apps for it to work?",
-                                "anchor": "check_in_luca_",

```

Comment:
The anchor here is actually missing the word "both" at the end, however, I still included this in the redirects, to make sure that even if this incomplete anchor link floats around somewhere on the internet, a useful response is shown.

Reply:
I'm not sure that is necessary, since this incorrect anchor `check_in_luca_` was only introduced into the English page when the luca Q&As were split up in the new FAQ design, so it hasn't been present for very long. I would ignore this error, but that is just my personal preference.

Reply:
I have no strong feelings in any direction, so I would leave it to the Open-Source-Team to decide whether this should also be a redirect or not.

Reply:
Wrong anchors have now been addressed in this PR and should be fixed